packages/playground/src/explorer/components/node_details.vue

```diff
@@ -99,35 +114,56 @@ export default {
     PublicConfigDetailsCard,
   },
 
-  setup(props) {
+  setup(props, { emit }) {
     const loading = ref(false);
     const dialog = ref(false);
+    const isError = ref(false);
+    const errorMessage = ref("");
     const route = useRoute();
     const node = ref<GridNode>(nodeInitializer);
 
-    watch(
-      () => props.nodeId,
-      async (nodeId: number) => {
-        if (nodeId > 0) {
+    function closeDialog(newValue: boolean) {
+      isError.value = false;
+      errorMessage.value = "";
+      emit("close-dialog", newValue);
+    }
+
+    async function requestNode() {
+      if (props.nodeId > 0) {
+        try {
           loading.value = true;
```

Comment:
I think we should set `isError.value` to `false` here in case the user clicks on `try again` to reset it